arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -446,13 +420,14 @@
       },
       "required": [
         "name"
-      ]
+      ],
+      "description": "The parameters used to check the availabity of the storage account name."
     },
     "CheckNameAvailabilityResult": {
       "properties": {
         "nameAvailable": {
           "type": "boolean",
-          "description": "Gets a boolean value that indicates whether the name is available for you to use. If true, the name is available. If false, the name has already been taken or invalid and cannot be used."
+          "description": "Gets a boolean value that indicates whether the name is available for you to use. If true, the name is available. If false, the name has already been taken or is invalid and cannot be used."
```

Comment:
There MUST not be `Gets` or `Sets` in the description because that denotes description very specifically targeted towards C#.  Please update the description here